muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/profile/body/application/profile-info-and-settings.tsx

```diff
@@ -13,6 +13,7 @@ import moment from '~/lib/moment';
 import UpdateAddressDialog from '../../dialogs/update-address';
 import UpdateUsernamePasswordDialog from '../../dialogs/update-username-password';
 import Button from '~/components/general/button';
+import { runInThisContext } from 'vm';
```

Comment:
This doesn't belong here.

Reply:
Removed, I wonder where the heck that came from. It's my addition but I have no idea why it has been added in the first place.